### ISSUE TITLE
建议统一下编码.

### ISSUE THREAD

**Issue by Ldream** [Yukong/Deeplearning.ai-Solutions#4]
[('05.About this Course.srt', 1, 'utf-8'), 
('02.What is a neural network.srt', 1, 'utf-8'), 
('03.Supervised Learning with Neural Networks.srt', 1, 'utf-8'), 
('01.Welcome.srt', 1, 'utf-8'), 
('04.Why is Deep Learning taking off.srt', 1, 'utf-8'),
 ('06.Course Resources.srt', 1, 'UTF-8-SIG')]

[('02.Logistic Regression.srt', 1, 'UTF-8-SIG'),
 ('10.Gradient Descent on m Examples.srt', 1, 'UTF-8-SIG'),
 ('04.Gradient Descent.srt', 1, 'GB2312'),
 ('07.Computation graph.srt', 1, 'UTF-8-SIG'),
 ("14.Vectorizing Logistic Regression's Gradient Output.srt", 1, 'utf-8'), 
('16.A note on python_numpy vectors.srt', 1, 'utf-8'), 
('03.Logistic Regression Cost Function.srt', None, 'UTF-16LE'),
 ('18.Explanation of logistic regression cost function (optional).srt', 1, 'UTF-8-SIG'), 
('01.Binary Classification.srt', None, 'UTF-16LE'), 
('13.Vectorizing Logistic Regression.srt', 1, 'UTF-8-SIG'), 
('17.Quick tour of Jupyter_iPython Notebooks.srt', 1, 'GB2312'), 
('11.Vectorization.srt', 1, 'utf-8'), 
('15.Broadcasting in Python.srt', 1, 'utf-8'),
 ('12.More Examples of Vectorization.srt', 1, 'GB2312'),
 ('06.More Derivative Examples.srt', 1, 'GB2312'),
 ('09.Logistic Regression Gradient Descent.srt', 1, 'UTF-8-SIG'),
 ('08.Derivatives with a Computation Graph.srt', 1, 'utf-8'), 
('05.Derivatives.srt', 1, 'GB2312')]

**Comment by cycleuser** [comment]
对，这个主意好，可以使用 Python 写一个脚本，依次读取文件，然后保存成统一编码的。

**Comment by 0xMashiro** [comment]
已经统一成 utf-8 无BOM格式，不同翻译人员使用的文本编辑器默认设置可能不一样
